src/pages/test-report/components/toolbar/toolbar.scss

```diff
@@ -1,7 +1,7 @@
 toolbar {
   #wrapper {
     height: 68px;
-    padding: 12px;
+    padding: 12px 24px 12px 24px;
```

Comment:
You could use vertical/horizontal shorthand here.

Reply:
true, done